specification/storage/data-plane/Microsoft.BlobStorage/preview/2019-12-12/blob.json

```diff
@@ -9158,7 +9158,7 @@
         "FieldQuote",
         "RecordSeparator",
         "EscapeChar",
-        "HasHeaders"
+        "HeadersPresent"
```

Comment:
Has this been released? If so, this would be a breaking change.

Reply:
no, this is still in development.